### DISCUSSION TITLE
Is this browser work with VPN?

### DISCUSSION THREAD

**Discussion by gg0fox** [minbrowser/min#2394] [Q&A]
I can't access internet through vpn with this browser on my Mac.

**Comment by PalmerAL** [comment]
It should work. Can you give more details about your VPN configuration? Have you confirmed it works in Chrome?

**Reply by gg0fox** [reply]
Any other browser (Safari Chrome Brave) works just fine with ClashX Pro, which is set rule mode on my Mac. It seems only Min is having trouble going through my vpn network.

**Reply by JaafarMehrez** [reply]
Same thing for me, have you found any solutions?

**Reply by PalmerAL** [reply]
@JaafarMehrez Same questions for you - are you using the same software as the original reporter? Does the software have any configuration options that could be relevant?

**Reply by JaafarMehrez** [reply]
I have tried with two different softwares and different VPN providers, the first one is called XiYou (not much configuration available from the interface), and the other one is shadowsocksX-NG-R8 (set on global mode), in both cases Min is not able to capture the configuration which is not the case for any of the other browsers on Mac machine.

**Comment by janxkoci** [comment]
Min works fine with my university VPN (Fortinet VPN). Using it on Debian 12.
